bundles/org.openhab.binding.monopriceaudio/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,541 @@
+			<channel id="bass" typeId="bass"/>
+			<channel id="balance" typeId="balance"/>
+			<channel id="dnd" typeId="dnd"/>
+			<channel id="page" typeId="page"/>
+			<channel id="keypad" typeId="keypad"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone15">
+		<label>Zone 15</label>
+		<description>The Controls for Zone 15</description>
+		<channels>
+			<channel id="power" typeId="system.power"/>
+			<channel id="source" typeId="source"/>
+			<channel id="volume" typeId="system.volume"/>
+			<channel id="mute" typeId="system.mute"/>
+			<channel id="treble" typeId="treble"/>
+			<channel id="bass" typeId="bass"/>
+			<channel id="balance" typeId="balance"/>
+			<channel id="dnd" typeId="dnd"/>
+			<channel id="page" typeId="page"/>
+			<channel id="keypad" typeId="keypad"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone16">
+		<label>Zone 16</label>
+		<description>The Controls for Zone 16</description>
+		<channels>
+			<channel id="power" typeId="system.power"/>
+			<channel id="source" typeId="source"/>
+			<channel id="volume" typeId="system.volume"/>
+			<channel id="mute" typeId="system.mute"/>
+			<channel id="treble" typeId="treble"/>
+			<channel id="bass" typeId="bass"/>
+			<channel id="balance" typeId="balance"/>
+			<channel id="dnd" typeId="dnd"/>
+			<channel id="page" typeId="page"/>
+			<channel id="keypad" typeId="keypad"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone17">
+		<label>Zone 17</label>
+		<description>The Controls for Zone 17</description>
+		<channels>
+			<channel id="power" typeId="system.power"/>
+			<channel id="source" typeId="source"/>
+			<channel id="volume" typeId="system.volume"/>
+			<channel id="mute" typeId="system.mute"/>
+			<channel id="treble" typeId="treble"/>
+			<channel id="bass" typeId="bass"/>
+			<channel id="balance" typeId="balance"/>
+			<channel id="dnd" typeId="dnd"/>
+			<channel id="page" typeId="page"/>
+			<channel id="keypad" typeId="keypad"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone18">
+		<label>Zone 18</label>
+		<description>The Controls for Zone 18</description>
+		<channels>
+			<channel id="power" typeId="system.power"/>
+			<channel id="source" typeId="source"/>
+			<channel id="volume" typeId="system.volume"/>
+			<channel id="mute" typeId="system.mute"/>
+			<channel id="treble" typeId="treble"/>
+			<channel id="bass" typeId="bass"/>
+			<channel id="balance" typeId="balance"/>
+			<channel id="dnd" typeId="dnd"/>
+			<channel id="page" typeId="page"/>
+			<channel id="keypad" typeId="keypad"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="allon">
+		<item-type>Switch</item-type>
+		<label>All On</label>
+		<description>Turn All Zones On</description>
+	</channel-type>
+
+	<channel-type id="alloff">
+		<item-type>Switch</item-type>
+		<label>All Off</label>
+		<description>Turn All Zones Off</description>
+	</channel-type>
+
+	<channel-type id="source">
```

Comment:
Is there a reason why you don't specify min/max?

Reply:
I guess I thought you didn't need that when constrained by a drop down.

Reply:
The user could use Setpoint in the site map. Then, it'll be limited.